icubam/backoffice/templates/operational-dashboard.html

```diff
@@ -18,7 +18,7 @@ <h1 class='text-center'>{{ _("Operational dashboard") }}</h1>
 	   {% if current_region_name != 'All regions' %}
 	     {{ _("Region") }}:
            {% end %}
-	   {{ current_region_name }}
+	   {{ _(current_region_name) }}
```

Comment:
This is more debatable. I believe the region name does not need to be translated, does it?

Reply:
I agree. The reason of this l18n was to be able to localize "Toutes les régions" but giving it a second thought that could be made in the server handler. Will be updated.